test/index.html

```diff
@@ -126,6 +128,17 @@
         });
       }
 
+      const viewer = function(results) {
```

Comment:
This is a function that takes in a `results` param. This params is the results strings in the GraphiQL results page. 
The plugin function can process the results string and return whatever UI it want's rendered.